app/models/customer.py

```diff
@@ -1,4 +1,10 @@
 from app import db
 
 class Customer(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime())
+    videos = db.relationship("Video", secondary="rental", backref="customers")
```

Comment:
At the moment, any of these columns can contain `null` values.  Consider revisiting each column to determine which column should truly have `null` values versus not. For example, is it useful to store a `Customer` without a `name` or `phone number`? Those two columns can be considered _required_ information to store about a customer, despite our requirements not strictly describing that behavior.